### ISSUE TITLE
Header repeat is not possible to turn off

### ISSUE THREAD

**Issue by droberts-ctrlo** [PhilterPaper/PDF-Table#77]
We have found that when using header_props=>{repeat} and setting it to zero, the code resets it to the default (see Table.pm:L338), rather than giving the expected behaviour.

I have created a fix which we are using in our local version, but would like to offer a PR in order to mitigate and fix this.

**Comment by PhilterPaper** [comment]
Is this what you were thinking of for a fix?
```
337.          # Check other parameters and put defaults if needed
338.          $header_props->{'repeat'   } //= $repeat_default;
```
If that does the job, I can have a new release out in a few days (I have some other minor changes to make).

**Comment by droberts-ctrlo** [comment]
> ```
> 337.          # Check other parameters and put defaults if needed
> 338.          $header_props->{'repeat'   } //= $repeat_default;
> ```

Was actually thinking `$header_props->{'repeat'} = $repeat_default if !defined $header_props->{'repeat'}`

**Comment by PhilterPaper** [comment]
Should be doing the same thing. If that fixes it for you, I'll have a new release out later this week.

**Comment by droberts-ctrlo** [comment]
`$header_props->{'repeat'} ||= $repeat_default` means "If header props is undefined, or it is 0, set it to the default" whereas `$header_props->{'repeat'}=$repeat_default if !defined $header_props->{'repeat'}` will allow you to set the repeat parameter to 0 - also there's a bug when trying to set compatibility to the "old" functionality, because that setting is not defined in the allowed parameters list. :smile:

**Comment by PhilterPaper** [comment]
There's a difference between ||= (the old, erroneous code) and //=. Are you confusing the two?

**Comment by PhilterPaper** [comment]
Fixed in 1.006 release.
